Makefile

```diff
@@ -53,9 +53,9 @@ endif
 
 override CLUSTERS_ARGS += $(CLUSTER_SETTINGS_FLAG)
 override DEPLOY_ARGS += $(CLUSTER_SETTINGS_FLAG)
-override E2E_ARGS += $(CLUSTER_SETTINGS_FLAG)
+override E2E_ARGS += $(CLUSTER_SETTINGS_FLAG) cluster1 cluster2
 export DEPLOY_ARGS
-override UNIT_TEST_ARGS += cmd internal/cli internal/env internal/log
+override UNIT_TEST_ARGS += test internal/env
```

Comment:
These `Makefile` and `e2e.sh` changes are intentional/necessary? They don't seem to be covered by the commit message, or at least it's not clear to me.

Reply:
They're necessary. Adding "test" to UNIT_TEST_ARGS excludes the E2E tests from being executed by `make unit`. We do this in the other repos. I removed `internal/cli` and `internal/log` from exclusion b/c there are no unit tests in those dirs anyway - I'm unclear why they were added here in first place.

Reply:
That makes sense, thanks @tpantelis.

> I'm unclear why they were added here in first place

That was my main concern, didn't want to lose something there. Thanks for clarifying!